ebb_lint/test/test_flake8.py

```diff
@@ -400,6 +400,30 @@ class C(object):
 $L204$'a' 'b'  $L303$# noqa
 
     ''',
+
+    # Oh boy these following two cases need explanation. The prefix of the
+    # 'spam' node in both of these cases ends with a literal \ followed by a
+    # newline, which cannot be tokenized by generate_tokens because it expects
+    # something to follow that sequence. This is one hell of a special case,
+    # but it made flake8 fail hard on valid syntax. The first case is a prefix
+    # containing no comment, and the second case also includes a comment which
+    # requires scanning.
```

Comment:
@jayvdb This is actually using `lib2to3.pgen2.tokenize.generate_tokens`. I don't think it's a bug in `generate_tokens`, though; it is, in fact, an incomplete document. The only reason I'm using `generate_tokens` at all is to find comment tokens.